api/client/network.go

```diff
@@ -14,6 +10,10 @@ import (
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/filters"
 	"github.com/docker/engine-api/types/network"
+	"net"
+	"sort"
+	"strings"
+	"text/tabwriter"
```

Comment:
IMO, for the consistent coding style, these lines should after `fmt`. 
The order should be `golang pkg` and then `vendor pkg`